_pages/bigquery_schema.md

```diff
@@ -122,6 +122,63 @@ The following fields are deprecated and no longer have meaning in the data set.
 | `string`      |  `Ip_Address`            |
 | `bool`        |  `TruthValue`            |
 
-## Query Examples
+### Blacklist Flags Field
+
+An additional field called ```blacklist_flags``` was added to all tables in response to a "switch discard issue", which identified that some historical tests were affected by traffic microbursts. See our [blog post]({{ site.baseurl }}/blog/traffic-microbursts-and-their-effect-on-internet-measurement) for complete details.
+
+The ```blacklist_flags``` field is used to mark test results if they meet specific parameters that qualify them for inclusion or exclusion in queries or research analyses. This field was created to mark tests affected by the "switch discard issue" identified in 2015-2016, but M-Lab may use the field for other use cases in the future.
```

Comment:
mark test results _that could be impacted by site configuration issues, or otherwise communicate potentially relevant information about the state of the platform at the time of the test_.